.github/workflows/preview.yml

```diff
@@ -45,7 +45,10 @@ jobs:
         run: echo "PR_DATA=$(cat ./pr.json)" >> $GITHUB_ENV
 
       - name: Download other artifacts
-        run: gh run download {{ github.event.worfklow_run.id }} -n 'foreman-docs-html-${{ fromJSON(env.PR_DATA).branch_name }}' -n foreman-docs-html-base -n foreman-docs-web-master
```

Comment:
I'm blind and I think this should be `${{ github.event.worfklow_run.id }}`. Then you can probably still download multiple artifacts at the same time.

Reply:
https://github.com/theforeman/foreman-documentation/pull/3587 tries that.